src/components/Gigs/Summary.svelte

```diff
@@ -1,34 +1,33 @@
 <script>
-  import moment from "moment-timezone";
   import Signup from "./Signup.svelte";
   import TooltipText from "../TooltipText.svelte";
   import Lineup from "./Lineup.svelte";
   import { writable } from "svelte/store";
   import { themeName } from "../../view";
+  import { stores } from "@sapper/app";
+  import { DateTime, Settings } from "luxon";
+  Settings.defaultZoneName = "Europe/London";
+
+  const suffix = (n) =>
```

Comment:
why is this duplicated from above? can it be shared?

Reply:
yes